plugin/metrics/prometheus/options.go

```diff
@@ -63,9 +76,24 @@ func NewOptions(primaryNamespace string) *Options {
 // AddFlags from this storage to the CLI.
 func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 	nsConfig := &opt.Primary
-	flagSet.String(nsConfig.namespace+suffixServerURL, defaultServerURL, "The Prometheus server's URL, must include the protocol scheme e.g. http://localhost:9090")
-	flagSet.Duration(nsConfig.namespace+suffixConnectTimeout, defaultConnectTimeout, "The period to wait for a connection to Prometheus when executing queries.")
-	flagSet.String(nsConfig.namespace+suffixTokenFilePath, defaultTokenFilePath, "The path to a file containing the bearer token which will be included when executing queries against the Prometheus API.")
+	flagSet.String(nsConfig.namespace+suffixServerURL, defaultServerURL,
+		"The Prometheus server's URL, must include the protocol scheme e.g. http://localhost:9090")
+	flagSet.Duration(nsConfig.namespace+suffixConnectTimeout, defaultConnectTimeout,
+		"The period to wait for a connection to Prometheus when executing queries.")
+	flagSet.String(nsConfig.namespace+suffixTokenFilePath, defaultTokenFilePath,
+		"The path to a file containing the bearer token which will be included when executing queries against the Prometheus API.")
+
+	flagSet.Bool(nsConfig.namespace+suffixSupportSpanmetricsConnector, defaultSupportSpanmetricsConnector,
+		`Whether if queries should be adjusted for OpenTelemetry Collector's spanmetrics connector.'`)
+	flagSet.String(nsConfig.namespace+suffixMetricNamespace, defaultMetricNamespace,
+		`The metric namespace that is prefixed to the metric name. A '.' separator will be added between `+
+			`the namespace and the metric name.`)
+	flagSet.String(nsConfig.namespace+suffixLatencyUnit, defaultLatencyUnit,
```

Comment:
why not flagSet.Duration?

Reply:
I like the thinking behind the idea, but it means users need to configure this value to `1ms` or `1s`, which seems a little awkward to me. `ms` or `s` results in a parse error.

Reply:
are ok, fair enough